pkg/controller/util.go

```diff
@@ -633,33 +633,43 @@ func MakeCloneSourcePodSpec(image, pullPolicy, sourcePvcName string, pvc *v1.Per
 			},
 		},
 		Spec: v1.PodSpec{
-			Containers: []v1.Container{
+			// We create initContainer just to set the pod as privileged.
+			// The pod has to be privileged as it has to have access to the hostPath in the node.
+			// However, currently there is a bug that we cannot attach block device to the pod if the pod (container)
+			// is privileged:
+			//https://github.com/kubernetes/kubernetes/issues/58251
+			//https://github.com/kubernetes/kubernetes/issues/62560
+			// As a reulst of that, instread setting the SecurityContext field as Privileged, in the main conatiner
+			// where we need to have access to  the hostPath, we set the SecurityContext field in the initContainer,
+			// and that will do it.
+			InitContainers: []v1.Container{
 				{
-					Name:            common.ClonerSourcePodName,
-					Image:           image,
-					ImagePullPolicy: v1.PullPolicy(pullPolicy),
-					SecurityContext: &v1.SecurityContext{
-						Privileged: &[]bool{true}[0],
-						RunAsUser:  &[]int64{0}[0],
-					},
-
+					Name:  "myinit",
+					Image: "registry.hub.docker.com/library/busybox:latest",
```

Comment:
Any other suggestions for a better image?

Reply:
You can use the same image as the cloner pod.  Just set the container Command/Args params appropriately.

I'd also recommend you name the container `init` instead of `myinit`.